index.js

```diff
@@ -45,14 +45,16 @@ function reporter(context, options = {}) {
 						const [matched, matchedWord] = match;
 
 						if (alternative) {
-							const replacement = matched.replace(
-								matchedWord,
-								cloneCase(alternative, matchedWord)
-							);
-							const range = [index, index + matched.length];
-							const fix = fixer.replaceTextRange(range, replacement);
-							const message = `Avoid using “${word}”, use “${alternative}” instead`;
-							report(node, new RuleError(message, { index, fix }));
+							if (alternative !== matchedWord) {
+								const replacement = matched.replace(
+									matchedWord,
+									cloneCase(alternative, matchedWord)
+								);
+								const range = [index, index + matched.length];
+								const fix = fixer.replaceTextRange(range, replacement);
+								const message = `Avoid using “${matchedWord}”, use “${alternative}” instead`;
```

Comment:
I'm using `matchedWord` instead of `word` otherwise the report is confusing when we want to check the capitalization.
For instance, given the following definition: `Asciidoctor > Asciidoctor` and the sentence: `AsciiDoctor is great` it will report:

```
Avoid using “Asciidoctor”, use “Asciidoctor” instead
```


With this change it will report:


```
Avoid using “AsciiDoctor”, use “Asciidoctor” instead
```